src/Views/Admin/Pages/Reports.php

```diff
@@ -114,4 +122,21 @@ public function get_all_time_start()
 
         return isset($donations[0]) ? $donations[0]->date : $start->format('Y-m-d H:i:s');
     }
+
+    /**
+     * @unreleased
+     */
+    private function getDismissedRecommendation(): string
+    {
+        $dismissedOption = '';
+        $option = get_option('givewp_reports_recurring_recommendation_dismissed', false);
+        $recurringAddonIsActive = '';
```

Comment:
Looks like this is never anything but empty?

Reply:
Got it. It's been updated to include fee recovery Resolved: https://github.com/impress-org/givewp/pull/6796/commits/804dca421dfc3f82a8c9cc76e4d96b7a983960f0#diff-9b924a759f61b0bf131cd15ca66e0984ccc0b166086c08a59b1b8519477eb4f7